traits/trait_types.py

```diff
@@ -225,19 +226,13 @@ def create_editor(self):
 
 
 class Int(BaseInt):
-    """ Defines a trait whose type must be an integer using a C-level fast
-        validator.
+    """ Defines trait that holds an int and uses a C-level fast validator.
```

Comment:
The wording of "integer" rather than "int" the original docstring was deliberate, since `Int` accepts integer-like things (most importantly, NumPy integers) as well as plain `int`s.  Maybe we should spell this out more fully? Something like: "must be an int or integer-like object", with a fuller explanation of what exactly "integer-like" means elsewhere.